python/delta-kernel-rust-sharing-wrapper/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "delta-kernel-python"
+name = "delta-kernel-rust-sharing-wrapper"
 edition = "2021"
 license = "Apache-2.0"
-version = "0.0.0"
+version = "0.0.1"
```

Comment:
let's still test with 0.0.1 and in the actual release we'll use 0.1.0

Reply:
makes sense